fhir-server/src/main/java/com/ibm/fhir/server/operation/spi/FHIROperationContext.java

```diff
@@ -59,74 +59,79 @@
      * associated with the REST API request for which the interceptor is being invoked.
      */
     public static final String PROPNAME_SECURITY_CONTEXT = "SECURITY_CONTEXT";
-    
+
     /**
-     *This property is of type HttpMethod 
+     * This property is to inject the JWT into the Operation Context.
+     */
+    public static final String PROPNAME_JWT = "JWT";
```

Comment:
missed this before, but I don't think we need the JWT in the context.  we already have the pertinent info from that in the SecurityContext.  additionally, if someone really needed it, its available via the Authorization header in the request context as well.